support-frontend/assets/helpers/campaigns/campaigns.ts

```diff
@@ -2,81 +2,84 @@ import type { TickerSettings } from 'components/ticker/contributionTicker';
+  copy?: (goalReached: boolean) => CampaignCopy;
+  formMessage?: JSX.Element;
+  termsAndConditions?: (
+    contributionsTermsLink: string,
+    contactEmail: string,
+  ) => JSX.Element;
+  cssModifiers?: string[];
+  contributionTypes?: ContributionTypes;
+  backgroundImage?: string;
+  extraComponent?: JSX.Element;
+  tickerSettings?: TickerSettings;
+  goalReachedCopy?: JSX.Element;
+  // If set, the form will be replaced with this if goal reached
+  createReferralCodes: boolean;
 };
 
-const currentCampaignPath: string | null = 'au/contribute';
+const currentCampaignPath: string | null = 'us/contribute';
 
-const aus2021Copy = (): CampaignCopy => ({
-	headerCopy: 'Support Guardian Australia with a contribution of any size',
-	contributeCopy:
-		'Together, we can be a voice for change. Help us reach our ambitious goal of 170,000 supporters.',
+const usEoy2021Copy = (): CampaignCopy => ({
+  headerCopy: "Join us in the fight for America's future",
+  contributeCopy:
+    "Quality, independent journalism that is freely accessible to all has never been more crucial. We’re raising $1.25m to fund our reporting in 2022. If you can, support the Guardian today.",
 });
 
 export const campaign: CampaignSettings = {
-	campaignCode: 'Aus_moment_2021',
-	copy: aus2021Copy,
-	tickerSettings: {
-		tickerCountType: 'people',
-		tickerEndType: 'unlimited',
-		currencySymbol: '$',
-		copy: {
-			countLabel: 'supporters in Australia',
-			goalReachedPrimary: "We've hit our goal!",
-			goalReachedSecondary: 'but you can still support us',
-		},
-	},
-	createReferralCodes: false,
+  campaignCode: 'Us_eoy_2021',
+  copy: usEoy2021Copy,
+  tickerSettings: {
+    tickerCountType: 'money',
+    tickerEndType: 'unlimited',
+    currencySymbol: '$',
+    copy: {
+      countLabel: 'contributed',
+      goalReachedPrimary: "We've hit our goal!",
+      goalReachedSecondary: 'but you can still support us',
+    },
+  },
+  createReferralCodes: false,
 };
 
 function campaignEnabledForUser(
-	campaignCode: string | null | undefined,
+  campaignCode: string | null | undefined,
 ): boolean {
-	if (currentCampaignPath && window.guardian.enableContributionsCampaign) {
-		return (
-			window.guardian.forceContributionsCampaign ||
-			window.location.pathname.endsWith(`/${currentCampaignPath}`) ||
-			campaign.campaignCode === campaignCode
-		);
-	}
+  if (currentCampaignPath && window.guardian.enableContributionsCampaign) {
 
-	return false;
+    return (
+      window.guardian.forceContributionsCampaign ||
+      window.location.pathname.endsWith(`/${currentCampaignPath}`) ||
+      campaign.campaignCode === campaignCode
+    );
+  }
+
+  return false;
 }
 
 export function getCampaignSettings(
-	campaignCode?: string,
+  campaignCode?: string,
 ): CampaignSettings | null {
-	if (campaignEnabledForUser(campaignCode)) {
-		return campaign;
-	}
+  if (campaignEnabledForUser(campaignCode)) {
+    console.log('campaignEnabledForUser: ', campaignEnabledForUser(campaignCode))
```

Comment:
Sneaky console log, one slipped by me a little while back, am thinking we could add in a pre-commit check

Reply:
Thanks @marialani!